nixos/modules/services/misc/gitea.nix

```diff
@@ -242,7 +250,8 @@ in
 
       enableUnixSocket = mkOption {
         type = types.bool;
-        default = false;
+        default = if cfg.configureNginx then true else false;
```

Comment:
What's wrong with `default = cfg.configureNginx` here?

Reply:
Allows to automatically switch to using a socket if nginx configuration is active.

Reply:
My point is, `configureNginx` is a boolean, so `if cfg.configureNginx then true else false` is equivalent to `cfg.configureNginx`.